test/config.test.js

```diff
@@ -1192,6 +1194,64 @@ describe(`configTest: ${printPath("[test/config.test.js]")}`, function () {
         }
     });
 
+    it("testing that the debug mode is set", async function () {
```

Comment:
this is not how you test if the logging is working.. you need to actually listen to the console and see if it outputs in there?

Reply:
Done. overrode the `debug.log` to capture logs.